corehq/apps/dump_reload/sql/serialization.py

```diff
@@ -3,19 +3,53 @@
 
 from django.core.serializers.json import Serializer as JsonSerializer
 
+from corehq.apps.hqwebapp.const import PASSWORD_PLACEHOLDER
 from corehq.util.json import CommCareJSONEncoder
 
 
 class JsonLinesSerializer(JsonSerializer):
     """
     Convert a queryset to JSON outputting one object per line
     """
+
+    def serialize(
+        self,
+        queryset,
+        *,
+        stream=None,
+        fields=None,
+        use_natural_foreign_keys=False,
+        use_natural_primary_keys=False,
+        progress_output=None,
+        object_count=0,
+        reset_encrypted_fields=True,
+        **options
+    ):
+        self.reset_encrypted_fields = reset_encrypted_fields
+        super().serialize(
+            queryset,
+            stream=stream,
+            fields=fields,
+            use_natural_foreign_keys=use_natural_foreign_keys,
+            use_natural_primary_keys=use_natural_primary_keys,
+            progress_output=progress_output,
+            object_count=object_count,
+            **options
+        )
```

Comment:
This is all just to support passing in `reset_encrypted_fields` which we realistically won't change, but demonstrates that you could use JsonLinesSerializer with this setting disabled if needed.

Reply:
Could you not use `*args, reset_encrypted_fields=True, **kwargs` to avoid having to specify all the others?

Also, since this is never currently set to 'False' can't we just leave the config param out for now and always do it?

Reply:
> Also, since this is never currently set to 'False' can't we just leave the config param out for now and always do it?

Yeah I tried to clarify this in my previous comment, but I only did this to make it clear that this serializer has this option, but I could be convinced that that is overkill.

Reply:
+1 to leaving it out for now. The presence of the `encrypted_fields` attribute is adequate to signal that encrypted fields should get special treatment.

Reply:
a16fc511d0b025189289253970dccf1839c241c6